src/deluge/gui/views/automation_view.cpp

```diff
@@ -121,7 +121,7 @@ const uint32_t mutePadActionUIModes[] = {UI_MODE_NOTES_PRESSED, UI_MODE_AUDITION
 
 const uint32_t verticalScrollUIModes[] = {UI_MODE_NOTES_PRESSED, UI_MODE_AUDITIONING, UI_MODE_RECORD_COUNT_IN, 0};
 
-constexpr int32_t kNumNonGlobalParamsForAutomation = 63;
+constexpr int32_t kNumNonGlobalParamsForAutomation = 64;
```

Comment:
Can you update the automation view documentation also please? The number and list of parameters is also in that document, thanks